pkg/expr/nodes.go

```diff
@@ -254,8 +254,14 @@ func (dn *DSNode) Execute(ctx context.Context, vars mathexp.Vars, s *Service) (m
 			}
 		}
 
+		dataSource := dn.datasource.Type
 		for _, frame := range qr.Frames {
 			logger.Debug("expression datasource query (seriesSet)", "query", refID)
+			// Check for TimeSeriesTypeNot in an InfluxDB query.
+			// This check should be removed once inconsistencies in data source responses are solved.
+			if frame.TimeSeriesSchema().Type == data.TimeSeriesTypeNot && dataSource == models.DS_INFLUXDB {
+				continue
```

Comment:
Do we know anything more specific about the shape of the frame (the fields) that we want to drop that we could could check for and/or comment on?

Reply:
Also perhaps log a warning here? Seems like we don't want to silently drop, that way if there are future issues with influx, that logging information can be used to see if this is being hit.